Helper/FeatureSwitch/Definitions.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Bolt magento2 plugin
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Bolt
+ * @package    Bolt_Boltpay
+ * @copyright  Copyright (c) 2018 Bolt Financial, Inc (https://www.bolt.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+namespace Bolt\Boltpay\Helper\FeatureSwitch;
+
+/**
+ * All feature switches are defined here. By default all new switches should be
+ * false and not rolled out. These values are only fallbacks for when something
+ * isn't defined on the Bolt side. Anything values coming from Bolt take presedence.
+ *
+ * Class Definitions
+ * @package Bolt\Boltpay\Helper\FeatureSwitch
+ */
+class Definitions {
+    const NAME_KEY="name";
+    const VAL_KEY="value";
+    const DEFAULT_VAL_KEY="default_value";
+    const ROLLOUT_KEY="rollout_percentage";
+
+    const M2_SAMPLE_SWITCH_NAME = "M2_SAMPLE_SWITCH";
```

Comment:
can you add comment what is this (is it example switch for testing)?

Reply:
yes this is until someone adds some switch we will actually use. it used to test this feature overall :)